### DISCUSSION TITLE
使用Nuxt3搭配ElementPlus时，组件出现红色波浪线

### DISCUSSION THREAD

**Discussion by tendecade** [element-plus/element-plus#12579] [Q&A]
<img width="1516" alt="image" src="https://user-images.githubusercontent.com/73530157/233885951-7502d053-6226-40ca-9902-f3b759faaa90.png">

代码中所有使用ElementPlus组件的地方，都报了上述错误；
上午代码还是正常的，下午再看的时候代码就开始报红了；

`
"nuxt": "^3.3.1"
"element-plus": "^2.3.1"
"@element-plus/icons-vue": "^2.1.0"
"@element-plus/nuxt": "^1.0.4"
`

**Comment by yeonjulee1005** [comment]
my code also same ! who has anyone answer it? 😢 
<img width="933" alt="image" src="https://user-images.githubusercontent.com/57179957/233899940-9952067a-7c19-4e17-bef8-6a858f880176.png">

`my package versions below`

```
"nuxt": "3.4.2"
"element-plus": "2.3.4"
"@element-plus/nuxt": "^1.0.4"
"@element-plus/icons-vue": "^2.1.0"
```

**Reply by tendecade** [reply]
I haven't found any relevant answers yet

**Reply by yeonjulee1005** [reply]
hello my friend! 😄 
i found something!! 
i seems like bug occur in "Typescript Vue Plugin (Volar)" 
it cant recognized hyphen tag 😢 

please see below link 😄 
https://github.com/vuejs/language-tools/issues/2688

it will be fix released edge version

**Reply by tendecade** [reply]
Good afternoon, friend～
I also found the discussion results of this previous issue in the discussion area https://github.com/element-plus/element-plus/discussions/3259 '; And I have already solved this problem.
